### DISCUSSION TITLE
Upgrade Restsharp108

### DISCUSSION THREAD

**Discussion by abuzuhri** [abuzuhri/Amazon-SP-API-CSharp#352] [General]
Upgrade Restsharp108 

Thanks @J-W-Chan for your work to Upgrade its look ok now after test please submit new PR and I will merge to main 

https://github.com/J-W-Chan/Amazon-SP-API-CSharp/tree/updateToRestsharp108 

we going to have beta version first then release after more testing , @RenzoF Please help us for full test once its ready

**Comment by abuzuhri** [comment]
Here we go https://www.nuget.org/packages/CSharpAmazonSpAPI/1.5.0-alpha1

alpha version ready for test , hopefully we get more developer test before go live

**Comment by RenzoF** [comment]
A quick test on this I noticed that 
`amazonConnection.Messaging.GetMessagingActionsForOrder(`
Returns all properties null using the 1.5.0-alpha package and expected results if I use the 1.4.32 package.

Will do more testing next week, though we might need some unit testing maybe?

**Reply by abuzuhri** [reply]
Please share sample code for that .

**Reply by RenzoF** [reply]
For the failing test?, is literally one line and takes and order number easy to replicate. Has anyone tested this or just made sure that it builds?

**Reply by abuzuhri** [reply]
I go live with change please keep watching any new Issue

**Comment by abuzuhri** [comment]
how going here 
i see many download alpha version and no body report any problem , for **GetMessagingActionsForOrder** i cant try this because i don't has permission in my market place to this 

can you check that
